civiform/civiform
# TI sub-banner announcing which client they're applying for

### Description

Replaced the nav bar "(Applying for lastName, firstName)" text for trusted intermediaries with the following banner.  The banner also contains a link for the TI to return back to the TI dashboard.  The sub-banner should appear when the TI is applying on behalf of a client, but should not appear on the TI dashboard page.

![image](https://github.com/civiform/civiform/assets/134556723/15c61c8a-d46b-4218-be47-fc962c300608)

## Release notes

Replaced the nav bar "(Applying for lastName, firstName)" text for trusted intermediaries with a sub-banner which states who they are applying for and offers a link to return to the TI dashboard. 

### Checklist

#### General


#### User visible changes


### Instructions for manual testing

1.  Log in as a TI.
2. On the TI dashboard, choose a client from the client list and click on the "Applicant Dashboard->" link.
3. Note the sub-banner that tells you which client you're applying for.
4. Apply for a program and note that the sub-banner appears on the program summary page and the application block pages.
5. Click on "Select a new client" to be redirected to the TI dashboard.  Note that the banner does not appear on that page.

### Issue(s) this completes

Fixes #5431
